docs/conf.py

```diff
@@ -28,7 +28,9 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = []
+extensions = [
+    'sphinx.ext.graphviz'
```

Comment:
I get the following error when trying to build:

```
WARNING: dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting
```

Looks like graphviz isn't installed in the docker container? Or the build isn't happening in a container?